ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -1855,6 +1856,7 @@ objects:
       - -c
       - |
         #!/bin/bash
+        set -euo pipefail
```

Comment:
Maybe not fail here... as teardown can have failures but should attempt to try to completion.

Reply:
the teardown script has the following protections that make this ok i think:

1) `trap 'teardown' EXIT`
2)
```
       function teardown() {
          set +e
```

Reply:
@jstuever agree?

Reply:
hold on.  teardown shouldn't need to set +e.  We have a "correct" teardown on most things - if we have a non standard teardown it would only apply in that context of one job (i.e. don't go adding set +e to teardown that don't have it already)